src/utils/json.js

```diff
@@ -120,3 +120,15 @@ function hasOwnProperty(obj, key) {
     key
   );
 }
+
+/**
+ * Converts the passed string into a JSON object (if possible) and returns the
+ * value of the propertyName on that object.
+ * @param {string} jsonString
+ * @param {string} propertyName
+ * @return {*}
```

Comment:
How about we return String or null?

@return {?string}

Reply:
Because that would be a lie.  An object can contain anything.  For instance:
fun(" {val: true} ", "val") 

would return true which is not a string.